node/src/chain_spec/darwinia.rs

```diff
@@ -188,7 +188,7 @@ pub fn genesis_config() -> ChainSpec {
 				treasury: Default::default(),
 
 				// Utility stuff.
-				sudo: Default::default(),
+				sudo: SudoConfig { key: Some(array_bytes::hex_n_into_unchecked(ALITH)) },
```

Comment:
This reminds me of the `AssetsConfig` set in the state processor, we might not need them there.